spec/models/service/retirement_management_spec.rb

```diff
@@ -96,15 +99,13 @@
   end
 
   it "with one src_id" do
-    User.current_user = user
-    expect(ServiceRetireRequest).to receive(:make_request).with(nil, {:src_ids => ['yabadabadoo'], :__request_type__ => "service_retire"}, User.current_user, true)
-    @service.class.to_s.demodulize.constantize.make_retire_request('yabadabadoo', User.current_user)
```

Comment:
thank you so much for removing this `'yabadabadoo'` junk.
I like the Flinstones and all but... yea